### DISCUSSION TITLE
问题：我已经把软件下好了，但是没有服务器地址，我要从哪里获取。

### DISCUSSION THREAD

**Discussion by 2365663** [shadowsocks/shadowsocks-windows#3214] [Q&A]
前同事给的SS链接，升级到最新的版本以后，SS链接不能用并且显示timeout。请问获取新的服务器是不是需要我购买？有没有推荐的服务器的服务商？

**Comment by chenshaoju** [comment]
你好，此处只提供 Shadowsocks for Windows 软件本体的故障报告，至于服务，处于安全等多方面因素的原因，此处无法提供，请理解。

**Reply by 2365663** [reply]
感谢告知。

**Comment by Jason-Study** [comment]
该软件是需要付费使用吗？

**Comment by gtese** [comment]
wahahahaha~~~~

**Comment by Makitoid** [comment]
那就是蹄子过期了，你再买一个吧
[给你推荐一个](http://maojia.net/)
